.github/workflows/ci.yml

```
@@ -33,26 +33,50 @@ jobs:
+      - name: Evaluate lint and build status
         id: status
         run: |
-          result="failed"
-          message="빌드 실패 ❌"
-
-          if [ "${{ steps.build.outcome }}" == "success" ]; then
-            result="success"
-            message="빌드 성공 🎊"
+          lint_result="${{ steps.lint.outcome }}"
+          build_result="${{ steps.build.outcome }}"
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)